### ISSUE TITLE
User page: update profile button doesn't work for web sites with www subdomains

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy-fed#1206]
We're failing here, eg `authed user nature.com isn't object owner [] {}`:

https://github.com/snarfed/bridgy-fed/blob/f34d1b43a3b32b82e09cba101251be4484bbb23a/models.py#L937-L942

Full example log:

```
Running receive task 27737075331792088871
Params: [('authed_as', 'nature.com'), ('obj', 'ahBicmlkZ3ktZmVkZXJhdGVkch8LEgZPYmplY3QiE2h0dHBzOi8vbmF0dXJlLmNvbS8M')]
Receiving web person https://nature.com/ AS1: {
  "objectType": "person",
  "id": "https://www.nature.com"
  "displayName": "Nature",
  "summary": "Nature is the foremost international weekly scientific journal in the world and is the flagship journal for Nature Portfolio. It publishes the finest ...",
  "url": "https://www.nature.com",
  "urls": [{ "value": "https://www.nature.com" }],
  "image": [
    { "url": "https://www.nature.com/static/images/favicons/nature/favicon-48x48-b52890008c.png" },
    { "url": "https://www.nature.com/static/images/favicons/nature/favicon-32x32-3fe59ece92.png" },
    { "url": "https://www.nature.com/static/images/favicons/nature/favicon-16x16-951651ab72.png" },
    { "url": "https://www.nature.com/static/images/favicons/nature/favicon.ico" }
  ],
}
Key('MagicKey', 'www.nature.com') use_instead => Key('MagicKey', 'nature.com')
Wrapping in update: {
  "objectType": "activity",
  "verb": "update",
  "id": "https://nature.com/#bridgy-fed-update-2024-07-24T04:36:22.656467+00:00",
  "actor": "https://www.nature.com",
  "object": {
    "objectType": "person",
    "id": "https://www.nature.com"
    "displayName": "Nature",
    "summary": "Nature is the foremost international weekly scientific journal in the world and is the flagship journal for Nature Portfolio. It publishes the finest ...",
    "url": "https://www.nature.com",
    "urls": [{ "value": "https://www.nature.com" }],
    "updated": "2024-07-24T04:36:22.656467+00:00",
    "image": [
      { "url": "https://www.nature.com/static/images/favicons/nature/favicon-48x48-b52890008c.png" },
      { "url": "https://www.nature.com/static/images/favicons/nature/favicon-32x32-3fe59ece92.png" },
      { "url": "https://www.nature.com/static/images/favicons/nature/favicon-16x16-951651ab72.png" },
      { "url": "https://www.nature.com/static/images/favicons/nature/favicon.ico" }
    ],
  }
}
Key('MagicKey', 'www.nature.com') use_instead => Key('MagicKey', 'nature.com')
Returning 403: authed user nature.com isn't object owner [] {}
```

**Comment by snarfed** [comment]
On a semi-related note, web sites with www also aren't getting linked to their user pages in their profile footers. See eg https://indieweb.social/@mackuba.eu@web.brid.gy vs https://indieweb.social/@nature.com@web.brid.gy .

**Comment by snarfed** [comment]
Fixed in #1490
